packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.test.js

```diff
@@ -21,9 +21,11 @@ import * as track from '../../index.track';
 
 import { EDdgDensity } from '../../../../model/ddg/types';
 
+const defaultDensityFromLocalStorage = localStorage.getItem('selectedDensity');
```

Comment:
what's the meaning of reading from local storage in a unit test?

Reply:
I apologise for my inexperience in writing tests, because I haven't worked on writing tests before. I had thought since the default configuration has changed, the selected node while running the test could be different from the default, hence I though it would be better to use the default selected node when the component mounts for testing rather than just the hard coded EDdgDensity.PreventPathEntanglement node